### PR TITLE
Add return type in README example

### PR DESCRIPTION
**Pull request by kaishin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ for up to date installation instructions.
 ```swift
 import Swish
 
-func baseRequest(url url: String, method: RequestMethod) {
+func baseRequest(url url: String, method: RequestMethod) -> NSURLRequest {
   let url = NSURL(string: url)!
   let request = NSMutableURLRequest(URL: url)
 
-  request.HTTPMethod = method.rawValue
+  return request.HTTPMethod = method.rawValue
```
Comment (paulyoung) [inline README.md]: This looks a bit strange to me, what with having the assignment on the same line as the return statement.

Comment (gfontenot) [inline README.md]: yeah, I don't think this will work. There should be an added `return request` line instead.

Comment (kaishin) [inline README.md]: My bad... fixing.

**Comment by gfontenot** [comment]
Yep, makes sense. :+1:

**Comment by gfontenot** [comment]
@kaishin I'm squashing/merging this

**Pull request merged**